graphdatascience/tests/unit/test_error_handling.py

```diff
@@ -120,3 +128,18 @@ def test_nonexisting_util_endpoint(gds: GraphDataScience) -> None:
 def test_wrong_client_only_prefix(gds: GraphDataScience) -> None:
     with pytest.raises(SyntaxError, match="There is no 'gds.beta.model.get' to call"):
         gds.beta.model.get("model")
+
+
+def test_incompatible_server_version(runner: QueryRunner) -> None:
+    G = Graph("dummy", runner, ServerVersion(0, 0, 0))  # This version object is not relevant for the test
+    gds = GraphDataScience(runner)
+    gds._server_version = ServerVersion(2, 2, 0)  # Something later than 2.1.0
```

Comment:
need to use pytest.mark.parametrize to set the version

Reply:
done